### PR TITLE
Add helper to create CHANGELOG file

### PR DESCRIPTION
**Pull request by madtrick**
## Summary

Initialize a CHANGELOG file with the prelude from http://keepachangelog.com/

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -9,6 +9,17 @@ var writeFile = promisify(fs.writeFile);
...
 import buildChangelog from './builder'
 
+export function init (path) {
+  path = path || 'CHANGELOG.md';
+
+  return parse(
+    [
+      '# Change Log',
+      'All notable changes to this project will be documented in this file.'
+    ].join('\n')
```
Comment (johanneswuerbach) [inline src/index.js:20]: Maybe also an empty line after that?

File: test/test.js [hunk 2]
```diff
@@ -22,4 +24,46 @@ describe('changelog', function() {
...
+        return init(path).then(function () {
+          var changelog = readFileSync(readPath).toString();
+          expect(changelog).to.equal(prelude);
+        });
+      });
+    }
+
+    describe('when no path is given', function () {
+      beforeEach(function () {
+        process.chdir('/tmp');
```
Comment (johanneswuerbach) [inline test/test.js]: https://nodejs.org/api/os.html#os_os_tmpdir would be better here.

File: src/index.js (hunk 1 above)
Comment (madtrick) [inline src/index.js:20]: It's injected by default. See the `\n` at the end of this line https://github.com/contentful-labs/keepachangelog/pull/13/files#diff-c1129c8b045390789fa8ff62f2c6b4a9R34

But I'll an explicit empty line to make it more clear.

File: test/test.js (hunk 2 above)
Comment (madtrick) [inline test/test.js]: :+1:

Comment (madtrick) [inline test/test.js]: Done

File: src/index.js (hunk 1 above)
Comment (madtrick) [inline src/index.js:20]: Fixed

**Comment by geigerzaehler** [comment]
:ship:

**Pull request merged**